### ISSUE TITLE
PEP8 and coding styles

### ISSUE THREAD

**Issue by tovin07** [tenable/nessrest#95]
Hello,

I think that this repo can be much better if you guys add some coding conventions.
It will be much more clearer than current.

PEP8, for example.

Can I fork and PEP8 all the code?

Thanks!

**Comment by streckc** [comment]
I think it is worth a shot.  But I don't have much experience with PEP8 other than my scripts do not follow it too closely.... yet.

Use 'python3 -m pep8 *.py' for validation?

**Comment by tovin07** [comment]
Yeah, something like that. Or, you can use pylint or flake8 to validate. I think your code will much more clearer and readable.
